File: eFormAPI/Plugins/ItemsGroupPlanning.Pn/ItemsGroupPlanning.Pn/Infrastructure/Models/ItemListCasesPnRequestModel.cs
namespace ItemsGroupPlanning.Pn.Infrastructure.Models
{
    using System;

    public class ItemListCasesPnRequestModel
    {
        public string NameFilter { get; set; }
        public string Sort { get; set; }
        public int PageIndex { get; set; }
        public int Offset { get; set; }
        public bool IsSortDsc { get; set; }
        public int PageSize { get; set; }
        public int ListId { get; set; }
        public DateTime? DateFrom { get; set; }
        public DateTime? DateTo { get; set; }
    }
}

File: eform-client/src/app/plugins/modules/items-group-planning-pn/models/list/item-list-cases-pn-request.model.ts
export class ItemListCasesPnRequestModel {
  pageSize: number;
  sort: string;
  pageIndex: number;
  isSortDsc: boolean;
  offset: number;
  nameFilter: string;
  listId: number;
  dateTo: string;
  dateFrom: string;

  constructor() {
    this.sort = 'Id';
    this.isSortDsc = true;
    this.pageSize = 10;
    this.pageIndex = 1;
    this.offset = 0;
    this.nameFilter = '';
  }
}
